### ISSUE TITLE
`vagrant up` doesn't "just work", problem with base box

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse#3460]
I just installed Vagrant 1.8.7 and VirtualBox Version 5.1.8 r111374 on Mac OS X 10.11.6 (newish laptop), switched to "master" (7e5e8c9), and ran `vagrant up` but it doesn't "just work" as advertised at http://guides.dataverse.org/en/4.5.1/installation/prep.html#vagrant-for-testing-only

```
murphy:~ pdurbin$ cd github/iqss/dataverse
murphy:dataverse pdurbin$ git co master
Branch master set up to track remote branch master from origin.
Switched to a new branch 'master'
murphy:dataverse pdurbin$ vagrant up
OPERATING_SYSTEM environment variable not specified. Using centos by default.
To specify it in bash: export OPERATING_SYSTEM=debian
MAIL_SERVER environment variable not specified. Using localhost by default.
To specify it in bash: export MAIL_SERVER=localhost
Bringing machine 'standalone' up with 'virtualbox' provider...
==> standalone: Box 'puppet-vagrant-boxes.puppetlabs.com-centos-65-x64-virtualbox-puppet.box' could not be found. Attempting to find and install...
    standalone: Box Provider: virtualbox
    standalone: Box Version: >= 0
==> standalone: Box file was not detected as metadata. Adding it directly...
==> standalone: Adding box 'puppet-vagrant-boxes.puppetlabs.com-centos-65-x64-virtualbox-puppet.box' (v0) for provider: virtualbox
    standalone: Downloading: http://puppet-vagrant-boxes.puppetlabs.com/centos-65-x64-virtualbox-puppet.box
An error occurred while downloading the remote file. The error
message, if any, is reproduced below. Please fix this error and try
again.


murphy:dataverse pdurbin$ git log --oneline | head -1
7e5e8c9 Merge pull request #3395 from IQSS/v4.5.1
murphy:dataverse pdurbin$
```

**Comment by pdurbin** [comment]
It turns out this is a problem with Vagrant 1.8.7. Per https://github.com/sbgrid/data-capture-module/issues/7 my workaround was this:

`sudo mv /opt/vagrant/embedded/bin/curl /opt/vagrant/embedded/bin/curl.moved`

Now Vagrant is able to download the "box":

```
murphy:dataverse pdurbin$ vagrant up
OPERATING_SYSTEM environment variable not specified. Using centos by default.
To specify it in bash: export OPERATING_SYSTEM=debian
MAIL_SERVER environment variable not specified. Using localhost by default.
To specify it in bash: export MAIL_SERVER=localhost
Bringing machine 'standalone' up with 'virtualbox' provider...
==> standalone: Box 'puppet-vagrant-boxes.puppetlabs.com-centos-65-x64-virtualbox-puppet.box' could not be found. Attempting to find and install...
    standalone: Box Provider: virtualbox
    standalone: Box Version: >= 0
==> standalone: Box file was not detected as metadata. Adding it directly...
==> standalone: Adding box 'puppet-vagrant-boxes.puppetlabs.com-centos-65-x64-virtualbox-puppet.box' (v0) for provider: virtualbox
    standalone: Downloading: http://puppet-vagrant-boxes.puppetlabs.com/centos-65-x64-virtualbox-puppet.box
 ```

Passing to QA to raise awareness and to close.

**Comment by kcondon** [comment]
@pdurbin, should this be documented? Otherwise I will close.

**Comment by pdurbin** [comment]
@kcondon meh, you can just close it. I'm sure the Vagrant project will fix https://github.com/mitchellh/vagrant/issues/7969 before long and release a fix in Vagrant 1.8.8 or whatever version.
